src/languageProvider/locales/en_US.json

```diff
@@ -57,6 +57,7 @@
   "oracle.betStartTimeDisabledText": "The betting start time has not started yet.",
   "oracle.consensus": "Consensus Threshold {value}. This value indicates the amount of BOT needed to set the result.",
   "oracle.enterAmountDisabledText": "You have not entered a valid amount.",
+  "oracle.eventUnconfirmed": "This created Event is unconfirmed. You cannot interact with it until it is confirmed by the blockchain.",
```

Comment:
Please add corresponding vars in chinese json

Reply:
ran the script